frontend-web/webclient/app/Applications/Overview.tsx

```diff
@@ -57,6 +57,17 @@ function Applications(props: ApplicationsProps): JSX.Element {
         "Seqtk"
     ];
 
+    const excludeTools = [
+        "bedtools",
+        "cellranger",
+        "homer",
+        "kallisto",
+        "macs2",
+        "salmon",
+        "samtools",
+        "seqtk"
+    ]
+
```

Comment:
Just to make sure, these are lowercased in the backend as well, right?

Reply:
Yes they are, I found out that e.g. cellrangers tool name was cellranger while the tag name was Cell Ranger so there was a need for multiple lists. Most of the others worked by just lowercasing them on the backend.

Reply:
Is that a good approach?

Reply:
We general need a better approach than hard coding the tools and tags needed. Should perhaps be configurable from config file to the backend. But this might something that is moved to when we also redesign app tags.
This would also be a major rewrite of frontend code if it is the backend that determines which tools and tags will be shown.

Reply:
It's not so much the hardcoding as much as the uppercase/lowercase aspect of it for me. The inconsistencies between calls makes coding/debugging a bit more problematic.

Reply:
But this is a rewrite issue, more than a current iteration issue